examples/css/style.css

```diff
@@ -57,26 +60,28 @@ body {
 .scatterplot .erase {
   justify-content: center;
   align-items: center;
+  height: 30px;
+}
+.scatterplot .erase:focus a {
+  outline: 0;
+}
+.scatterplot .erase:hover .erase-fill-element,
+.scatterplot .erase:focus .erase-fill-element {
+  fill: #5a3ec8;
 }
 
 .scatterplot .erase svg {
   cursor: pointer;
 }
 
 .scatterplot .menuEl, .scatterplot .menuEl2 {
```

Comment:
In the original code from Accurat, the menu elements contained the class `menuEl` and `menuEl2`. We can change it to stay consistent with `kebab-cased` if that makes sense.